### PR TITLE
FindBugs parser fix

### PR DESCRIPTION
**Pull request by SunweiWang**

### CODE REVIEW

File: src/main/java/BlueTurtle/parsers/FindBugsXMLParser.java [hunk 1]
```diff
@@ -86,8 +89,49 @@
...
+					String absoluteFilePath = "";
+					
+					String path = "";
+					// continue go down the list of source path if the file does not exist.
+//					do {
+					
+						pathFront = srcList.item(k).getTextContent();
+		
+						// concatenate the source path with the class name.
+						String fileConcate = pathFront + "\\" + cN + ".java";
```
Comment (Michieldoesburg) [inline src/main/java/BlueTurtle/parsers/FindBugsXMLParser.java]: Can't you use File.separator() instead of \\?

**Comment by ClintonCao** [comment]
To solve the path problems, we are going to implement a class that can gather all the information of the class files. This way the parsers does not have to worry about the path problems anymore.

**Comment by boninggong** [comment]
image

**Comment by Michieldoesburg** [comment]
Bam.

**Pull request merged**